taxi/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
```

Comment:
No need to reassign, those are default values

Reply:
does not work without it

Reply:
Yess, it will be `user` by default, because of AbstractUser, sorry